BasicGUI/QMLSources/TransformatedPreview.qml

```diff
@@ -0,0 +1,11 @@
+import QtQuick 2.15
+import QtQuick.Window 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.3
+import QtQuick.Dialogs 1.3
+import Main 1.0
+
+Control
+{ 
+    
+}
```

Comment:
If this is a working state which is not necessary for this delivery then please keep this file only in local history or separate branch.

Reply:
Corrected